test/tetris/BoardTest.java

```diff
@@ -0,0 +1,39 @@
+package tetris;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+public class BoardTest {
```

Comment:
This project could use more tests — particularly of the rotation, motion, and collision detection.